app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,17 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
cái `||=` gọi là gì và dùng để làm gì vậy bạn?

Reply:
dùng để ghi nhớ kết quả của một hàm để tăng tốc ứng dụng của bạn khi mà hàm đó đã được tính toán.